pkg/kubectl/deployment.go

```diff
@@ -56,18 +74,37 @@ func (s DeploymentBasicGeneratorV1) Generate(params map[string]interface{}) (run
 	if !isArray {
 		return nil, fmt.Errorf("expected []string, found :%v", imageStrings)
 	}
-	delegate := &DeploymentBasicGeneratorV1{Name: name, Images: imageStrings}
-	return delegate.StructuredGenerate()
+	return &BaseDeploymentGenerator{
```

Comment:
nit: should the previous format string use "%T" instead of "%v"?  It is saying it expected some type, but then prints out the value it saw, not the type.